style.css

```diff
@@ -403,11 +403,131 @@ nav .hamburger-menu {
     margin-top: 5%;
   }
 
-  .works .card button {
+  .card .card-button {
     margin-top: 5%;
   }
 }
 
+/* Popup Card for Projects */
+
+.card-modal-container {
+  position: fixed;
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  overflow: auto;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(193, 199, 208, 0.7);
+}
```

Comment:
- [ ] Kindly make sure to make the background of the pop-up card `blur` so it can match the provided Figma design as much as possible.

    _**Hint: `backdrop-filter: blur(5px);`**_ _Please add this CSS property it will be perfect_


    ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/223854173-d11c5522-f663-4e33-9704-3254ff4769a8.png)

   ### FIGMA DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/223854063-faab272e-ebe2-4248-b349-8392f4d2c4c1.png)